streams/src/main/java/org/apache/kafka/streams/kstream/Suppressed.java

```diff
@@ -102,13 +102,6 @@ static StrictBufferConfig unbounded() {
          */
         StrictBufferConfig shutDownWhenFull();
 
-        /**
-         * Sets the buffer to use on-disk storage if it requires more memory than the constraints allow.
-         *
-         * This buffer is "strict" in the sense that it will never emit early.
-         */
-        StrictBufferConfig spillToDiskWhenFull();
```

Comment:
on-the-side cleanup. This feature isn't implemented yet, so we'll just remove it from the API for now.